app/controllers/api/v1/CommentsController.js

```diff
@@ -48,6 +48,12 @@ export default class CommentsController {
 
       const timelines = await newComment.create()
 
+      timelines.forEach(async (timeline) => {
+        if (timeline.isDirects()) {
+          await PubSub.updateUnreadDirects(timeline.userId)
+        }
+      })
```

Comment:
Hmm... I actually used `Promise.all(timelines.map(...)` in the first version of this change, but then decided to simplify it since it doesn't use returned values anyway. But if you are sure we need it, I'll bring it back, OK.